.travis.yml

```diff
@@ -3,4 +3,4 @@ node_js: 4
 before_script:
   - npm install -g gulp
 script:
-  - gulp ci-tests
+  - gulp ci-tests --ci
```

Comment:
Or rename ci-tests to just tests. So you can use `gulp tests` for testing locally and `gulp tests --ci` to test using travis.